docs/en/enterprise-edition/content-collections/application-security/ides/connect-vscode.adoc

```diff
@@ -40,9 +40,9 @@ NOTE: The plugin automatically invokes the latest version of Prisma Cloud Code S
 
 .. For *GRBAC*, provide the following permissions.
 +
-* *Policies*: 'View' permissions
-* *Projects*: 'View' permissions
-* *Repositories*: ('View' and 'Create') OR ('View' and 'Update') permissions
+* *Policies* -> *Policies*: 'View' permissions
```

Comment:
Why duplicate  "Policies"? 
State the entity requiring the permissions, the permissions required and then the path.

Reply:
One is the header, one is the line
<img width="865" alt="Screenshot 2024-10-27 at 10 43 41 PM" src="https://github.com/user-attachments/assets/189fe3e4-71c4-45f7-8110-a2aa28b2ed1c">